Makefile

```diff
@@ -276,26 +272,11 @@ test\:cli:
 test\:names:
 	gotestsum --format testname -- $(DEFAULT_TEST_DIRECTORIES) $(TEST_FLAGS)
 
-.PHONY: test\:lens
-test\:lens:
-	@$(MAKE) deps:lens
-	gotestsum --format testname -- ./$(LENS_TEST_DIRECTORY)/... $(TEST_FLAGS)
-
-.PHONY: test\:lens-quick
-test\:lens-quick:
-	@$(MAKE) deps:lens
-	gotestsum --format testname -- ./$(LENS_TEST_DIRECTORY)/...
-
 .PHONY: test\:all
 test\:all:
 	@$(MAKE) test:names
 	@$(MAKE) test:lens
```

Comment:
question: Do we still need this? Does `make test` not suffice?

Reply:
Will do - thanks Shahzad :)

- [ ] Remove `make test:all`